argo/README.md

```diff
@@ -0,0 +1 @@
+this is the legacy argo2 Solr index.  When argo2 was shut down in favor of argo3, this index became stale.
```

Comment:
I think if we call this file `admin-extra.html`, it'll show up in the solr admin too, which might be nice for people poking around the solr UI.